package.json

```
@@ -4,10 +4,11 @@
 	"private": true,
```

Manque le main pour que les scripts soient bien tous fonctionnels comme il faut
```suggestion
	"private": true,
	"main": "build/index.js",
```

---

Euh non ? Il n'y en a pas besoin

---

Si, parce que si tu fais "npm run start" sans cette ligne, il va essayer de lancer ./index.js (qui n'existe pas, évidemment), au lieu de ./build/index.js .
Si je l'ai fait remonter c'est parce que j'ai eu le "Cannot find module X" puisqu'il manquait cette ligne justement xD

---

Ah ok, moi j'utilise directement `node build/index.js`